src/core/lib/event_engine/cf_engine/cf_engine.cc

```diff
@@ -15,6 +15,7 @@
 #include <grpc/support/port_platform.h>
 
 #ifdef GPR_APPLE
+#if __MAC_OS_X_VERSION_MIN_REQUIRED > __MAC_10_12
```

Comment:
@HannahShiSFB Do `grpc` headers define `__MAC_10_12`? (If not, then this will not work correctly.)